libc/include/llvm-libc-macros/linux/watch-queue-macros.h

```diff
@@ -1,5 +1,4 @@
-//===-- Definition of macros from watch-queue.h
-//---------------------------------===//
+//===-- Definition of macros from watch-queue.h ---------------------------------===//
```

Comment:
`git clang-format` does not like this. Please ignore for now as this file may not even be required.

Reply:
That header is way too long, shorten it to 80 characters and it will stop doing this.